ios/Podfile

```diff
@@ -17,6 +17,7 @@ abstract_target 'Expo Go' do
   pod 'Google-Maps-iOS-Utils', :git => 'https://github.com/googlemaps/google-maps-ios-utils.git'
   pod 'JKBigInteger', :podspec => 'vendored/common/JKBigInteger.podspec.json'
   pod 'MBProgressHUD', '~> 1.2.0'
+  pod 'SocketRocket', '~> 0.6.0' # transitive dependency of React-Core and we use it to get the `RCTInspectorPackagerConnection` state
```

Comment:
I'm wondering if we should specify the version – doesn't React-Core already lock us to a specific version?